src/backend/kvm/config.rs

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use goblin::elf64::program_header::PT_LOAD;
+use sallyport::elf::pf::kvm::SALLYPORT;
+
+pub struct Config {}
+
+impl super::super::Config for Config {
+    type Flags = u32;
+
+    fn flags(flags: u32) -> Self::Flags {
+        flags
+    }
+
+    fn new(shim: &super::super::Binary, _exec: &super::super::Binary) -> Result<Self> {
+        if shim
+            .headers(PT_LOAD)
+            .filter(|p| p.p_flags & SALLYPORT != 0)
+            .count()
+            != 1
```

Comment:
Break this into two statements to avoid wrapping.

Reply:
yes